src/applications/personalization/profile/tests/e2e/address-validation/military-address.cypress.spec.js

```diff
@@ -0,0 +1,37 @@
+import { setUp } from 'applications/personalization/profile/tests/e2e/address-validation/setup';
+
+describe('Personal and contact information', () => {
+  describe('when entering a military address', () => {
+    it('should successfully update on Desktop', () => {
+      setUp('military');
+
+      cy.get('[type="checkbox"]')
+        .first()
+        .check();
+
+      cy.get('#root_addressLine1')
+        .clear()
+        .type('PSC 808 Box 37');
+      cy.get('#root_addressLine2').clear();
+
+      cy.findByLabelText(/City/i).select('FPO');
```

Comment:
this test seems to fail here since there is no `City` field once you select the military address checkbox.